OpenRA.Game/Actor.cs

```diff
@@ -22,7 +22,7 @@
 
 namespace OpenRA
 {
-	public sealed class Actor : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding, IEquatable<Actor>, IDisposable
+	public sealed class Actor : IScriptBindable, IScriptNotifyBind, ILuaTableBinding, ILuaEqualityBinding, ILuaToStringBinding, ISync, IEquatable<Actor>, IDisposable
```

Comment:
I'd prefer it if you could remove the [Sync] attribute from the owner, since it currently has no purpose except to be be misleading.